malaria-app-android/src/main/java/com/peacecorps/malaria/activities/RapidFireGame.java

```diff
@@ -236,17 +241,31 @@ public RapidFireTimeCounter(long millisInFuture, long countDownInterval) {
         @Override
         public void onTick(long l) {
             timer.setText(""+l/1000);
+            millisLeft=l;
         }
 
         @Override
         public void onFinish() {
             counter.cancel();
             prepNextQues();
         }
+    }
 
-
+    @Override
+    protected void onPause() {
+        counter.cancel();
+        counter= null;
+        super.onPause();
     }
 
+    @Override
+    protected void onResume() {
+        if(counter==null){
+            counter= new RapidFireTimeCounter(millisLeft,1000);
```

Comment:
@chhavip  CountDownTimer once cancelled can't be started again from the same point using the same object. We have to follow this approach only (Cancelling and then creating new Timer which will be given initial time as the time that was remaining i.e millisLeft) :)